components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/java/org/wso2/carbon/identity/mgt/endpoint/client/ApiClient.java

```diff
@@ -491,7 +491,10 @@ private ClientResponse getAPIResponse(String path, String method, List<Pair> que
 
         statusCode = response.getStatusInfo().getStatusCode();
         responseHeaders = response.getHeaders();
-
+        String responseBody = null;
+        if (response.hasEntity()) {
```

Comment:
to avoid java.io.IOException: stream is closed exception which occurred with the previous approach